test/conftest.py

```diff
@@ -48,6 +48,12 @@ def dirtyrepo(tmp_path):
         yield pygit2.Repository(path)
 
 
+@pytest.fixture
+def dirtyrepostatus(tmp_path):
+    with utils.TemporaryRepository('dirtyrepostatus.zip', tmp_path) as path:
```

Comment:
`dirtyrepostatus.zip` is copy of `dirtyrepo.zip` which also includes a `.gitignore` and an ignored file.

Reply:
If the new `dirtyrepostatus.zip` actually replaces `dirtyrepo.zip` will the test suite pass? If it does then better to keep only one zip (for the repo size to be smaller better than bigger).

Reply:
Didn't modify `dirtyrepo.zip` in place because I was afraid it might break tests. Will try running the test suite replacing `dirtyrepo.zip` and wil keep it that way if no tests break.

Reply:
It's kinda tricky to update `dirtyrepo.zip` without breaking other tests. The easiest solution seems to be adding a `.gitignore` to `dirtyrepo` (without committing it) and then modifying these two tests so that they pass:

```diff --git a/test/test_diff.py b/test/test_diff.py
index 55269ca..61007c6 100644
--- a/test/test_diff.py
+++ b/test/test_diff.py
@@ -64,6 +64,7 @@ index 297efb8..0000000
 PATCHID = 'f31412498a17e6c3fbc635f2c5f9aa3ef4c1a9b7'
 
 DIFF_HEAD_TO_INDEX_EXPECTED = [
+    '.gitignore',
     'staged_changes',
     'staged_changes_file_deleted',
     'staged_changes_file_modified',
@@ -87,6 +88,7 @@ DIFF_HEAD_TO_WORKDIR_EXPECTED = [
 ]
 
 DIFF_INDEX_TO_WORK_EXPECTED = [
+    '.gitignore',
     'file_deleted',
     'modified_file',
     'staged_changes_file_deleted',
```

Would this be ok?

Reply:
Yes